src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -0,0 +1,119 @@
+@snippet methodSampleDoc(docConfig)
+    @let ApiName = docConfig.getApiName, \
+            apiName = context.upperCamelToLowerCamel(ApiName), \
+            methodName = docConfig.getMethodName(), \
+            returnType = docConfig.getReturnType(), \
+            initCodeSpec = docConfig.getInitCode()
+        try {
+            ${@apiName} = new {@ApiName}();
+            {@initCode(initCodeSpec)}
+            @if docConfig.isPagedVariant()
+                for ({@methodCallSampleCode(methodName, apiName, initCodeSpec)} as element) {
```

Comment:
Yea, the Java one is sequential. I think that's the difference between two languages, in PHP the key() method is required by the Iterator interface: http://php.net/manual/en/class.iterator.php